source/common/protobuf/utility.cc

```diff
@@ -191,7 +191,9 @@ void deprecatedFieldHelper(Runtime::Loader* runtime, bool proto_annotated_as_dep
 #else
   bool warn_only = true;
 #endif
-
+  if (Runtime::runtimeFeatureEnabled("envoy.features.disallow_deprecated_features")) {
```

Comment:
Would it make sense to use the supplied `runtime` argument to this method? Not sure if that helps with testability etc, but easier to reason about one runtime instead of a global and local one.

Reply:
Done.